x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/support/ValidationTests.java

```diff
@@ -133,38 +133,40 @@ public class ValidationTests extends ESTestCase {
         '\r'
     );
 
+    private static final int MAX_NAME_LENGTH = Validation.MAX_NAME_LENGTH;
+
     public void testUsernameValid() throws Exception {
-        int length = randomIntBetween(Validation.MIN_NAME_LENGTH, Validation.MAX_NAME_LENGTH);
+        int length = randomIntBetween(Validation.MIN_NAME_LENGTH, MAX_NAME_LENGTH);
         String name = new String(generateValidName(length));
-        assertThat(Users.validateUsername(name, false, Settings.EMPTY), nullValue());
+        assertThat(Users.validateUsername(name, false, Settings.EMPTY, MAX_NAME_LENGTH), nullValue());
```

Comment:
Do we still have a test that is for this method signature?
```
Users.validateUsername(String, boolean, Settings)
```
If not, I think we should have one.

Reply:
We don't. I'm happy to add them.